src/Components/Tabs/Tabs.js

```diff
@@ -115,17 +116,20 @@ class Tabs extends Component {
   }
 
   getTabWidth(width) {
-    const { scrollEnabled, actionItems } = this.props;
+    const { scrollEnabled, actionItems, fixedTabWidth } = this.props;
+    const hasFixedTabWidth = !!fixedTabWidth;
+    let fixedWidth;
 
     if (!scrollEnabled) {
       let tabWidth = width / actionItems.length;
+      fixedWidth = fixedWidth * actionItems.length;
```

Comment:
You've defined `fixedWidth` above as undefined, then down here you're setting it equal to itself times the length of items. Should this be:
```
fixedWidth = fixedTabWidth * actionItems.length;
```